Kubernetes-Practical/Service/Service.yaml

```
@@ -0,0 +1,90 @@
+  name: nginx-service-clusterip
+spec:
+  selector:
+    app: nginx
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 80
+  type: ClusterIP
+
+
+### LoadBalancer Service 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)